.github/actions/set_up_macos/action.yml

```
@@ -5,20 +5,28 @@ inputs:
   repo_name:
     description: The name of the GitHub repo.
     required: true
-  xcode_version:
-    description: The version of Xcode to use.
-    default: '14.3.1'
+  test_target:
+    description: The test target to execute.
+    default: ''
 
 runs:
   using: composite
   steps:
     - uses: maxim-lobanov/setup-xcode@v1
       with:
-        xcode-version: ${{ inputs.xcode_version }}
+        xcode-version: '15.0.1'
+    - name: Override Xcode Version
+      shell: bash
+      run: |
+        # TODO: Investigate why these examples fail on CI with Xcode 15
```

```suggestion
        # GH762: Investigate why these examples fail on CI with Xcode 15
```
I try to keep TODO comments out of the checked-in code. Instead, I use `GHXXX` comments where the `XXX` is the issue number to track the follow-up work.